include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -50,6 +50,15 @@ class MGConstrainedDoFs : public Subscriptor
    * edges of a refinement level are treated corrected but leaves degrees of
    * freedom at the boundary of the domain untouched assuming that no
    * Dirichlet boundary conditions for them exist.
+   *
+   * Furthermore, this call sets up a ConstraintMatrix on each level that
```

Comment:
No, this should be a very cheap operation. Since we're only building simple 1-to-1 relations in the constraints, the cost is essentially that of setting up two vectors. Among all other things that happen in the transfer this should be pretty low cost. If we don't fill in any constraints, the data structure will only contain a single index set in the end which I think we have no trouble holding.